dll/cpl/input/input.c

```diff
@@ -96,9 +87,9 @@ LONG CALLBACK
 CPlApplet(HWND hwndCPl, UINT uMsg, LPARAM lParam1, LPARAM lParam2)
 {
     CPLINFO *CPlInfo;
-    DWORD i;
+    int i;
 
-    i = (DWORD)lParam1;
+    i = (int)lParam1;
```

Comment:
The variable `i` should be in `[0, CPL_GETCOUNT – 1]`.
(See `CPL_INQUIRE` and `CPL_DBLCLK`)
And `CPL_GETCOUNT` is one.
Thus, the variable `i` should be one.

Reply:
We still don't have a before-picture, that was shot with Katayamas code. Does it show the exact same icon or another one?

Reply:
@JoachimHenze the "before" picture was posted by Katayama in comments of recent commit: https://user-images.githubusercontent.com/2107452/189984046-b5c516ed-6a37-47ad-a40e-a0c79651cebc.png

There should be no visual difference between "before" and "after".

Reply:
Can you elaborate the difference?

Reply:
@JoachimHenze 

> We still don't have a before-picture, that was shot with Katayamas code. Does it show the exact same icon or another one?

Katayama's code works, the result is the same.
This PR is from my analysis and comparison with other CPLs.

Reply:
Other CPLs don't have to set the small icon manually and works just fine with `int`.

So `LoadImage` code is a hack?

Reply:
> So LoadImage code is a hack?

That was improper code.

Reply:
I think it would be better to set the icon the same way as other CPLs do.